command/agent/config-test-fixtures/basic.hcl

```diff
@@ -190,3 +191,17 @@ autopilot {
 	server_stabilization_time = "23057s"
 	enable_custom_upgrades = true
 }
+plugin "docker" {
```

Comment:
Do we ever need to worry about naming collisions here between plugin types?

Reply:
`plugin "docker" {` just seems a little too abstract as to whats going on.

Reply:
That is just the binary name inside the plugins folder. We may have to think about how to resolve a conflict when trying to configure a builtin plugin when there is a conflict. This is just a parsing starting point. The fields in the config can evolve as we figure it out